### PR TITLE
Feature: Add Vault item UI component

### PR DESCRIPTION
**Pull request by baomastr**

### CODE REVIEW

File: src/components/v2/VaultItemUi/styles.ts [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+      justify-content: space-between;
+      margin-bottom: ${theme.spacing(6)};
+    `,
+    title: css`
+      display: flex;
+      align-items: center;
+    `,
+    tokenIcon: css`
+      width: 20px;
+      height: 20px;
```
Comment (therealemjy) [inline src/components/v2/VaultItemUi/styles.ts:23]: We should use `theme.iconSize` for these

File: src/components/v2/VaultItemUi/styles.ts [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+      margin-right: ${theme.spacing(1)};
+    `,
+    tokenIconLarge: css`
+      width: 32px;
+      height: 32px;
```
Comment (therealemjy) [inline src/components/v2/VaultItemUi/styles.ts:28]: We should use `theme.iconSize` for these

File: src/components/v2/VaultItemUi/styles.ts [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    textAligned: css`
+      display: inline-flex;
+      align-items: center;
+    `,
+    rewardWrapper: css`
+      display: flex;
+      align-items: center;
+
+      ${theme.breakpoints.down('sm')} {
+        font-size: 14px;
```
Comment (therealemjy) [inline src/components/v2/VaultItemUi/styles.ts:57]: We can get this font size from the typography variants in the theme (`small1` or `small2` would work).

File: src/components/v2/VaultItemUi/index.tsx [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+      {
+        title: t('vaultItemUi.stakingApr', { stakeTokenName: getToken(tokenId).symbol }),
+        value: formatToReadablePercentage(stakingAprPercentage),
+      },
+      {
+        title: t('vaultItemUi.dailyEmission'),
+        value: (
+          <>
+            <Icon css={classes.tokenIcon} name={rewardTokenId} />
+            {formatCommaThousandsPeriodDecimal(
```
Comment (therealemjy) [inline src/components/v2/VaultItemUi/index.tsx:61]: You don't need to use `formatCommaThousandsPeriodDecimal` here, `convertWeiToCoins` already uses it internally.

File: src/components/v2/VaultItemUi/index.tsx [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+            )}
+          </>
+        ),
+      },
+      {
+        title: t('vaultItemUi.totalStaked'),
+        value: (
+          <>
+            <Icon css={classes.tokenIcon} name={tokenId} />
+            {formatCommaThousandsPeriodDecimal(
```
Comment (therealemjy) [inline src/components/v2/VaultItemUi/index.tsx:76]: You don't need to use `formatCommaThousandsPeriodDecimal` here, `convertWeiToCoins` already uses it internally.

**Review by therealemjy — COMMENTED** [review]
Nice, it looks good 👍
A few more comments and we should be good

File: src/components/v2/VaultItemUi/styles.ts (hunk 3 above)
Comment (baomastr) [reply src/components/v2/VaultItemUi/styles.ts:57]: this component has custom sizes that don't fit our typography styles.

File: src/components/v2/VaultItemUi/styles.ts (hunk 2 above)
Comment (baomastr) [reply src/components/v2/VaultItemUi/styles.ts:28]: done

File: src/components/v2/VaultItemUi/styles.ts (hunk 1 above)
Comment (baomastr) [reply src/components/v2/VaultItemUi/styles.ts:23]: done

**Comment by baomastr** [comment]
moved to https://github.com/VenusProtocol/venus-protocol-interface/pull/548 due to rebase conflicts

**Pull request closed without merging**